### ISSUE TITLE
No default Super Game Boy border

### ISSUE THREAD

**Issue by rtomasa** [libretro/mgba#251]
This is not a bug but a feature request to display the default Super Game Boy bezel on GB/GBC games when Super Game Boy option is enabled and the games are not SGB enhanced. Currently the core displays a solid color instead of the bezel or black color border.

Also if mgba_gb_model = "Autodetect" the bezels won't be set for many games. You must start the core with the option already set to "Super Game Boy". Same happens to all configurations that do not take effect until you close retroarch and init again.

**Comment by endrift** [comment]
The lack of default border should now be fixed with a custom border.

Also most GB games don't have SGB borders. Are the games you're testing SGB enhanced?

**Comment by rtomasa** [comment]
Yes, I'm talking specifically about games that are not SGB Enhanced. If you have the SGB option enabled, these games are displayed with a color bezel right now.
What I suggest is to display the default SGB bezel as in the original hardware which is a DMG like bezel (https://youtu.be/NDuFFSZQwRI?t=208) or have an option to display it in black instead of the solid color.
